finagle-redis/src/test/scala/com/twitter/finagle/redis/commands/hyperloglog/HyperLogLogClientIntegrationSuite.scala

```diff
@@ -0,0 +1,35 @@
+package com.twitter.finagle.redis.integration
+
+import com.twitter.finagle.redis.naggati.RedisClientTest
+import com.twitter.finagle.redis.tags.{ClientTest, RedisTest}
+import com.twitter.util.{Future, Await}
+
+import org.junit.Ignore
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+@Ignore
```

Comment:
Yes, we're not currently running the finagle-redis integration tests in Pants. This `@Ignore` has no effect on the SBT build, but these tests are also excluded by default in `Build.scala`.

It would be nice to have them run as proper integration tests in SBT. I've just created [an issue](https://github.com/twitter/finagle/issues/360) to track this.